templates/cloud_config.j2

```diff
@@ -24,6 +24,26 @@ write_files:
     permissions: 0744
     content: |
       {{ lookup('file', role_path + '/files/healthcheck.sh') | indent(6, False) }}
+
+  {% if wimpy_aws_cloudwatch_enabled -%}
+  - path: /home/core/cloudwatch.conf
+    owner: core:core
+    permissions: 0744
+    content: |
+      log_group = "/{{ wimpy_deployment_environment }}/{{ wimpy_project_name }}"
```

Comment:
CloudWatch configuration that will save a log group for every application, so you can read logs for individual instances or aggregate for a given application.